courses/programming-in-haskell/pih-exercises/src/Chapter06.hs

```diff
@@ -0,0 +1,41 @@
+-- | Exercises for chapter 5.
+
+module Chapter06 (and', concat', replicate', (!!!), elem', merge, msort) where
```

Comment:
In the solutions branch you can see how to avoid name clashes with the prelude functions by using the `import ... hiding` construct ;)